TrinityCore/TrinityCore
# Core/Spells: Fix dispelling movement impairing auras by PvP-trinket and similar spells.

**Changes proposed:**

After [3306a4d](https://github.com/TrinityCore/TrinityCore/commit/3306a4d06cc557967c23adcc60c2e) spells that grant immunity to mechanics do not completely remove the auras that these mechanics have on only one of their effects. Instead, these effects are only temporarily disabled.

However, there seem to be exceptions to this. For example, PvP-trinkets, or Human racial "Every Man for Himself". 
Such spells are distinguished by the duration of their aura - only 100 ms. Of course, they should completely remove the auras they grant immunity to, and not just disable some effects for 100 milliseconds.

How to reproduce:

1. Target yourself and use `.aura 31589`
2. Use PvP-trinket or "Every Man for Himself"
3. See that nothing worked as expected

This PR fixed the issue.

**Tests performed:**

Builded and tested in-game.
